### ISSUE TITLE
Revert Changes To The Stable One

### ISSUE THREAD

**Issue by Linaqruf** [Linaqruf/kohya-trainer#232]
I'm sorry, it has been 2 months since the latest update, and everything has become complicated. It makes me uncomfortable to even look at this repository. I want to revert the changes before things became complicated. Regarding the /dev/ branch, here's what I want to do:

Things I want to do:
- Revert support for multi-concept training: This was never on my list because it's really hard to do in Colab. Yet, I still did it, and now we have been dealing with bugs for 2 months that I couldn't fix.
- Revert `recursive` arguments and `sub_folder` for certain cells: This also supports multi-concept training.
- Take back my word to simplify cells: It's hard to maintain simplified and automated processed cells. It also makes the training process inflexible, whereas my notebook promises flexibility. Since there is a 1-click Lora training notebook, I think I don't need to do this anymore.
- Remove the `Launch Portable Web UI` cell: This is my fault. I will focus on updating the inference cell next time.

Also, I hope you all can use [Kohya LoRA Finetuning](https://colab.research.google.com/github/Linaqruf/kohya-trainer/blob/main/kohya-LoRA-finetuner.ipynb) instead of [Kohya LoRA Dreambooth](https://colab.research.google.com/github/Linaqruf/kohya-trainer/blob/dev/kohya-LoRA-dreambooth.ipynb), so I can focus on one LoRA notebook. It's basically my fault from the start to create two LoRA notebooks.

**Comment by anothertal3** [comment]
Yes, I know that feeling when a code base gets out of control ;) 

I haven't used the "Lora Finetuning" yet and there are a couple of settings which are new to me (e.g., "Bucketing and Latent Caching"). There also doesn't appear to be as lot annotated as in the "Lora Dreambooth" notebook. Is there some "getting started" documentation to get familiar with the notebook? Or maybe a simple "migration" guide :)?


> Revert support for multi-concept training

Will it still be possible to change the `dataset.toml` or something similar by hand? I rarely use multi-concepts as such but I frequently use subsets to prioritize images (better images get more repetitions than those of lower quality).

**Comment by merodiro** [comment]
> Remove the Launch Portable Web UI cell

How can we test unsupported lora types such as `locon` and `loha` without the Web UI?
